src/App.tsx

```diff
@@ -15,71 +16,84 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+interface AppState {
+  selectedGood: string;
+}
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, AppState> {
+  state: AppState = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+  handleSelectGood = (good: string) => {
+    this.setState({ selectedGood: good });
+  };
+
+  handleClearSelection = () => {
+    this.setState({ selectedGood: '' });
+  };
+
+  render() {
+    const { selectedGood } = this.state;
+    const title = selectedGood ? `${selectedGood} is selected` : 'No goods selected';
+    const buttonClear = selectedGood && (
       <button
         data-cy="ClearButton"
         type="button"
         className="delete ml-3"
+        onClick={this.handleClearSelection}
+        aria-label="Clear selection"
       />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+    );
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        <h1 className="title">
+          {title}
+          {buttonClear}
+        </h1>
```

Comment:
The clear button is not aligned:
![image](https://user-images.githubusercontent.com/86923089/222958080-ebde2308-b75a-44a6-b4ad-f4715995334e.png)